src/js/components/BreadcrumbComponent.jsx

```diff
@@ -76,8 +165,16 @@ var BreadcrumbComponent = React.createClass({
   },
 
   render: function () {
+    var classSet = classNames({
+      breadcrumb: true,
+      collapsed: this.state.collapsed
+    });
+
+    // Hacky, but ensures that the dimension measurements are up-to-date
+    _.defer(this.updateDimensions);
```

Comment:
```_.defer(func)``` is similar to ```setTimout(func, 0)```.
This doesn't ensures that the component already rendered. It's only a thing of good luck.

Let's really think about a solid solution with ```componentDidUpdate```.